src/app/features/room-nav/RoomNavItem.tsx

```diff
@@ -217,9 +224,21 @@ export function RoomNavItem({
   const { focusWithinProps } = useFocusWithin({ onFocusWithinChange: setHover });
   const [menuAnchor, setMenuAnchor] = useState<RectCords>();
   const unread = useRoomUnread(room.roomId, roomToUnreadAtom);
+  const {
```

Comment:
Add: Indicate active call
```suggestion
  const {
    isCallActive,
```